### PR TITLE
Update implementation-plan.md

### PR DESCRIPTION
**Pull request by devoncarew**
- remove references to the Atom plugin and add references to the github grammar file

### CODE REVIEW

**Comment by devoncarew** [comment]
@lrhn - is there a template plan doc that we should also update?

**Comment by lrhn** [comment]
I don't actually know if there is a template file. I just took the file from an earlier change (maybe much earlier, which could account for discrepancies).

File: accepted/future-releases/small-features-21Q1/implementation-plan.md [hunk 1]
```diff
@@ -91,18 +91,10 @@ The analyzer should likely recognize the grammar even without the experiment fla
...
-this to handle the new syntax.
-
-[atom]: https://github.com/dart-atom/dart
-
-### VS Code
-
-Update the syntax highlighting grammar to support the control flow syntax (and,
-likely apply a very similar diff to the Atom grammar above).
+Update the (syntax highlighting grammar)[https://github.com/dart-lang/dart-syntax-highlight] to support
+the control flow syntax.
```
Comment (lrhn) [inline accepted/future-releases/small-features-21Q1/implementation-plan.md:97]: The "control flow syntax" is a legacy phrase I failed to replace.
Just make it "the new syntax".
(The syntax changes are in annotations and a shift operator).

**Review by lrhn — APPROVED** [review]

File: accepted/future-releases/small-features-21Q1/implementation-plan.md (hunk 1 above)
Comment (devoncarew) [reply accepted/future-releases/small-features-21Q1/implementation-plan.md]: Thanks! Updated.

**Review by parlough — APPROVED** [review]

**Comment by leafpetersen** [comment]
> @lrhn - is there a template plan doc that we should also update?

@devoncarew not for implementation plans, but templates/implementation_meta_issue.txt is intended to be the blueprint for the tracking issue.

**Comment by devoncarew** [comment]
> @devoncarew not for implementation plans, but templates/implementation_meta_issue.txt is intended to be the blueprint for the tracking issue.

Ah, cool. Everything there looks up to date (though I made some minor tweaks in https://github.com/dart-lang/language/pull/1436).

**Pull request merged**